src/Stratis.Bitcoin.Features.Api/Program.cs

```diff
@@ -36,7 +46,8 @@ public static IWebHost Initialize(IEnumerable<ServiceDescriptor> services, FullN
                     foreach (ServiceDescriptor service in services)
                     {
                         object obj = fullNode.Services.ServiceProvider.GetService(service.ServiceType);
-                        if (obj != null && service.Lifetime == ServiceLifetime.Singleton && service.ImplementationInstance == null)
+                        if (obj != null && service.Lifetime == ServiceLifetime.Singleton &&
```

Comment:
Brackets

Reply:
@monsieurleberre we don't have a maximum line length limit. I think up to 200 is acceptable.